Py_FS/wrapper/nature_inspired/MVO.py

```diff
@@ -0,0 +1,125 @@
+"""
+Programmer: Soumitri Chattopadhyay
+Date of Development: 27/07/2021
+This code has been developed according to the procedures mentioned in the following research article:
+"Mirjalili, S. and Mirjalili, S.M. and Hatamlou, A. Multi-Verse Optimizer: a nature-inspired algorithm for global optimization
+Neural Computing & Applications 27, 495–513 (2016)."
+"""
+
+import math
+
+import numpy as np
+from sklearn import datasets
+from sklearn import preprocessing
+
+from Py_FS.wrapper.nature_inspired.algorithm import Algorithm
+
+
+class MVO(Algorithm):
+
+    # Multi-Verse Optimizer
+    ############################### Parameters ####################################
+    #                                                                             #
+    #   num_agents: number of agents                                              #
+    #   max_iter: maximum number of generations                                   #
+    #   train_data: training samples of data                                      #
+    #   train_label: class labels for the training samples                        #
+    #   obj_function: the function to maximize while doing feature selection      #
+    #   trans_function_shape: shape of the transfer function used                 #
+    #   save_conv_graph: boolean value for saving convergence graph               #
+    #                                                                             #
+    ###############################################################################
+
+    def __init__(self,
+                 num_agents,
+                 max_iter,
+                 train_data,
+                 train_label,
+                 trans_func_shape='s',
+                 save_conv_graph=False,
+                 seed=0):
+
+        super().__init__(num_agents=num_agents,
+                         max_iter=max_iter,
+                         train_data=train_data,
+                         train_label=train_label,
+                         trans_func_shape=trans_func_shape,
+                         save_conv_graph=save_conv_graph,
+                         seed=seed)
+
+        self.algo_name = 'MVO'
+        self.agent_name = 'Universe'
+        self.algo_params = {}
+
+    def user_input(self):
+        self.algo_params['Min'] = 0.2
+        self.algo_params['Max'] = 1.0
+        self.algo_params['p'] = 0.6
+
```

Comment:
Allow the user to enter the values of the parameters [Refer to the user_input() method of WOA_test.py].

Reply:
Done, ranges/default values given as in relevant paper